tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/documentation/ErrorsTest.java

```diff
@@ -52,10 +52,37 @@ public void setup() throws IOException {
 
     @Test(description = "Test error type")
     public void testErrorAsAType() {
-        Assert.assertEquals(testModule.errors.size(), 2, "Two errors expected");
+        Assert.assertEquals(testModule.errors.size(), 5, "Two errors expected");
```

Comment:
Shall we say? `Five errors expected`

Reply:
Added the changes in a separate PR: https://github.com/ballerina-platform/ballerina-lang/pull/27873/files